src/scripts/main.js

```diff
@@ -1,3 +1,49 @@
 'use strict';
 
-// write code here
+const title = document.querySelectorAll('th');
+
+for (const titleName of title) {
+  titleName.addEventListener('click', (e) => {
+    const item = e.target;
+    const tbody = document.querySelector('tbody');
+    const rows = [...tbody.querySelectorAll('tr')];
+
+    if (item.innerText === 'Age' || item.innerText === 'Salary') {
+      let i = [...title].indexOf(item);
+
+      if (i === 6) {
+        i = 2;
+      }
+
+      if (i === 7) {
+        i = 3;
+      }
```

Comment:
why do you need this manipulation with `i`?

Reply:
to make footer to sort rows